slack_rules/slack_service_owner_transferred.yml

```diff
@@ -0,0 +1,81 @@
+AnalysisType: rule
+Filename: slack_service_owner_transferred.py
+RuleID: Slack.AuditLogs.ServiceOwnerTransferred
+DisplayName: Slack Service Owner Transferred
+Enabled: true
+LogTypes:
+  - Slack.AuditLogs
+Tags:
+  - Slack
+Severity: Critical
+Description: Detects transferring of service owner on request from primary owner
```

Comment:
I would remove the `on request from primary owner` from this description, there doesn't seem to be any labeling in the test log data for who the primary owner had been